### PR TITLE
[AT-3970] enable pylint for storage, host, and tools

### PR DESCRIPTION
**Pull request by seedlit**
Jira ticket [here.](https://up42.atlassian.net/browse/AT-3970)
Scope: enable pylint for all storage, tools, and host files

Items:
* [ ] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [ ] Bumped version
* [ ] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

**Review by dustgalactic — APPROVED** [review]

File: tests/fixtures/fixtures_storage.py [hunk 1]
```diff
@@ -1,63 +1,49 @@
...
-    storage = Storage(auth=auth_mock)
-
-    return storage
+    url_order_info = f"{constants.API_HOST}/v2/orders/{constants.ORDER_ID}"
+    requests_mock.get(url=url_order_info, json=constants.JSON_ORDER)
+    return storage.Storage(auth=auth_mock)
 
 
 @pytest.fixture()
 def storage_live(auth_live):
```
Comment (javidq) [inline tests/fixtures/fixtures_storage.py:48]: Not needed

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "up42-py"
-version = "0.37.0a5"
+version = "0.37.0a6"
```
Comment (javidq) [inline pyproject.toml:3]: To be updated

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,8 +30,13 @@ You can check your current version with the following command:
...
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
 
-## 0.37.0a5
+## 0.37.0a6
+
+**March 13, 2024**
 
+- Adjusted `host.py`, `tools.py`, `test_tools.py`, `storage.py`, `test_storage.py`, and `fixtures_storage.py` in accordance with Pylint checks.
+
+## 0.37.0a5
```
Comment (javidq) [inline CHANGELOG.md:39]: Master merge is needed

**Review by javidq — CHANGES_REQUESTED** [review]

**Review by janchrizz — APPROVED** [review]
for live tests, good to go:
- pystac.get_collection()
- storage.get_assets()
- storage.get_orders()

They are covered [here](https://github.com/up42/sdk-tests/blob/master/spec/test_storage.py)

File: pyproject.toml (hunk 2 above)
Comment (seedlit) [reply pyproject.toml:3]: I am a bit confused.
Which version should I update it to?

File: CHANGELOG.md [hunk 4]
```diff
@@ -41,7 +46,6 @@ For more information, see [UP42 Python package description](https://pypi.org/pro
 
 ## 0.37.0a4
 
-
```
[the remark this answers is not in the document]
Comment (seedlit) [reply CHANGELOG.md:44]: this was an extra line of space which was inconsistent with other entries in changelog.
Do you want me to bring it back?

File: pyproject.toml (hunk 2 above)
Comment (seedlit) [reply pyproject.toml:3]: I am a bit confused.
Which version should I update it to?

File: CHANGELOG.md (hunk 3 above)
Comment (seedlit) [reply CHANGELOG.md:39]: I think it already has the latest changes.
Am I missing something?

File: pyproject.toml (hunk 2 above)
Comment (javidq) [reply pyproject.toml:3]: Sorry, unmerged PRs confusion 😄

File: CHANGELOG.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (javidq) [reply CHANGELOG.md:44]: Motivation for change wasn't clear, thanks for clarification

File: CHANGELOG.md (hunk 3 above)
Comment (javidq) [reply CHANGELOG.md:39]: Sorry, my bad, confused due to diff visualization

**Review by javidq — APPROVED** [review]

**Pull request merged**